README.md

```diff
@@ -55,7 +58,7 @@ echo AVS_DIRECTORY_ADDRESS=$AVS_DIRECTORY_ADDRESS | sed 's/"//g' >> .env
 
 # Deploy Hello World AVS specific contracts
 (cd contracts &&\
-forge script script/HelloWorldDeployer.s.sol --rpc-url http://localhost:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 --broadcast --revert-strings debug )
+forge script script/HelloWorldDeployer.s.sol --rpc-url http://localhost:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 --broadcast)
```

Comment:
Hi @stevennevins - this is an interesting change. Could you share any context on why removing `--revert-strings debug` is a good idea? thx

Reply:
yeah i would reserve this for debugging, we can add it as a recommendation for some common errors or something in a troubleshooting section